source/2016-02-22-das-etwas-andere-lohnmodell.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: Das etwas andere Lohnmodell
+date: 2016-02-22 11:53 UTC
+author: bro
+tags:
+---
+
+Wer bei Panter arbeitet, weiss genau wie viel seine Arbeitskollegen verdienen - nämlich genau so viel, wie man selber verdient. Die Lohnhöhe variiert von Monat zu Monat und dies bei jedem. Einziger Unterschied, die zwei Lohnstufen. Weshalb es bei Panter keinen fixen Monatslohn gibt, die [Transparenz](http://blog.panter.ch/2015/11/10/transparenz-als-standard.html) auch beim Lohn gelebt wird und was es mit diesen zwei Lohnstufen auf sich hat, breiten wir hier gerne mal aus.
+
+
+## Panters monatliche Performance macht den Lohn
+Bei Panter variiert der Lohn von Monat zu Monat und wird mithilfe der monatlichen Performance und dem Stundensatz errechnet. Jeder Panter Mitarbeiter erfasst seine gesamten Arbeitsstunden pro Monat. Dabei wird unterschieden, ob die Stunden für ein Kundenprojekt geleistet wurden oder es sich um Panter interne Aufgaben gehandelt hat. Kurz; sind die gearbeiteten Stunden einem Kunden verrechenbar oder nicht?
+
+Aus all den Stunden wird die so genannte Performance berechnet: Anzahl aller verrechenbaren Stunden, geteilt durch alle erfassten Stunden aller Mitarbeiter.
+
+Der Stundensatz für Ferien wird jährlich neu festgelegt. Dieser wird anhand der durchschnittlichen Performance des Vorjahres berechnet. Der Lohn eines Mitarbeiters berechnet sich daher wie folgt:
+
+```
+Anzahl gearbeitete Stunden des betreffenden Mitarbeiters × Panters Monatsperformance × Stundensatz
```

Comment:
Würde fast behaupten hier darf man einen Apostroph verwenden um es vom Plural "Panters" zu unterscheiden ;-) Sonst einfach "monatliche Performance"?

Beim vorherigen Absatz ("Aus all...") würde ich noch was wie "Daraus ergibt sich ein Faktor zwischen 0 und 1 der unsere Rentabilität wiederspiegelt." anhängen damit klar ist was die Performance genau ist in Zahlenform.